modules/collectd/templates/etc/collectd/conf.d/etcd.conf.erb

```diff
@@ -76,7 +76,7 @@ stats_self = {
   ],
 }
 -%>
-  <URL "<%= @client_endpoint -%>/v2/stats/self">
+  <URL "http://<%= @client_endpoint -%>/v2/stats/self">
```

Comment:
is there a reason this changes to http? If so, can we add this in the commit message?